.github/workflows/wpt-updater.yml

```diff
@@ -11,6 +11,9 @@ concurrency:
   group: wpt-updater
   cancel-in-progress: true
 
+permissions:
+  contents: read
```

Comment:
This workflow commits and opens a pull request. Will this change break the workflow?

Reply:
Should be fine, as you have specific permission at the job level: https://github.com/ada-url/ada/blob/3113fafbc16a1de100b7919d35a2b6699966e8d8/.github/workflows/wpt-updater.yml#L20